example/main.go

```diff
@@ -78,26 +90,30 @@ func getAndUpdateItem(client *onepassword.Client, existingVaultID, existingItemI
 	if err != nil {
 		panic(err)
 	}
+	// [developer-docs.sdk.go.update-item]-end
 
 	for _, f := range updatedItem.Fields {
 		if f.Title == "Details" {
-			doSomethingSecret(f.Value)
+			fmt.Println(f.Value)
 		}
 	}
 }
```

Comment:
If we don't use this in our doc examples, whats the point of having this?

Reply:
I think we should include this in the docs example, to be consistent with the other SDKs, unless there's a reason to leave it out for Go that I'm not aware of

Reply:
@MOmarMiraj As opposed to the code snippets (which are only to exhibit a specific use-case/part of the SDK), I believe these examples should be a more comprehensive, complete usage guide.

I don't think this should be included within the same block because it dilutes the content - it's 5 more lines of code that do not bring any information to the table - I think that the essence of how to use `Put` is how to call the function and the expected output type, which we now capture suggestively with the variable naming. WDYT?

Reply:
My main concern is parity between the examples -- if we don't include printing the results here, could we also remove it from the other languages? That way we won't need to introduce the examples with different information in the docs.

Reply:
Yes, that makes sense I agree with that. I think its best to leave it in the example but not the docs.